cmd/login/command.go

```diff
@@ -9,11 +9,36 @@ import (
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
 )
 
 const (
-	name        = "login"
-	description = "Logs in into the Kubernetes cluster."
+	name             = "login [K8s API URL | Web UI URL | Existing GS Context Name]"
+	shortDescription = "Logs into an installation's Kubernetes API"
+	longDescription  = `Log into an installation's Kubernetes API.
+
+You can use as an argument:
+  * Your installation's Kubernetes API URL as an argument (e. g. 'https://g8s.test.eu-west-1.aws.gigantic.io')
```

Comment:
```suggestion
  * Your installation's Kubernetes API URL, e. g. 'https://g8s.test.eu-west-1.aws.gigantic.io'
```

Also I think our API host names usually start with `api.g8s.`.